content/admin/onboard_team/CreateUsers.md

```diff
@@ -1,44 +1,135 @@
+* Enter the user's **First Name**, **Last Name** and **Email Address** in the provided fields. 
+* Check the box if you want to **Require email activation** instead of requiring a password.
+* Choose which of the **System Roles** should apply to the user in the dropdown menu. The default is **None**. 
+* Choose the **Organization** to which the user belongs. 
+* Once you decide on an organization, you can choose the default **Organization Role** in the dropdown menu as well as an **Application Access Group** in the multiselect field.
+* Choose the **Date Format**, **Time Format** and **Time Zone** in the dropdown fields. 
+* The box to **Use Organization Settings** is checked by default. You can uncheck the box to create your own settings using the **Access** toggle or the box to **make user API only**. 
+* Click the **Add** button to save the user's information. 
 
->**Note:** Make sure you verify the [Role](admin-manageorgsroleperm.html#roles) so that users will have the privileges to carry out the actions you intend.
+<a href="assets/images/User-system-admin.png" rel="lightbox" title="Add a user as a Super Administrator"><img class="thumbnail" src="assets/images/User-system-admin.png"/></a>
 
-<a href="assets/images/Create_User.png" rel="lightbox" title="Add User"><img class="thumbnail" src="assets/images/Create_User.png"/></a>
 
-Users will receive an activation email to join Contrast. You will be able to check their status from the main Users table to identify who is waiting activation, active/inactive, or is locked out of their account based on security policy.
+### Multiple users 
+
+To bulk add users, click the upload icon above the grid in the **Users** page to import a spreadsheet with the users' information. The spreadsheet must be CSV format, and include the following fields for each user. **All field headings and values in the spreadsheet must be formatted as shown.** 
+
+* **First Name** 
+* **Last Name**
+* **Email** or **Username** <br> See the **Authentication** section below for more requirements. 
+* **Organization UUID** 
+* **Organization Role** <br> Values can be "View", "Edit", "Rules_admin" or "Admin".
+
+> **Note:** Find the Organization UUID in the Contrast interface by impersonating the appropriate organization, and then going to **Organization Settings > Organization tab > General Information section**. 
+
+##### Authentication 
+
+For users who have [HTTP Header](installation-setupauth.html#http-proxy), [LDAP authentication](installation-setupauth.html) or [AD](installation-setupauth.html#ad) authentication configured, you must use the field heading **Username** instead of **Email** in the spreadsheet. The username values entered in the spreadsheet and the authentication configuration must match exactly. 
+
+##### Templates
+
+Download **CSV templates** by hovering over the upload icon and clicking on the link, or by clicking the links below. 
```

Comment:
@kaitlynmortimer Suggestion: "Download the appropriate CSV template straight from the UI, or use the links below."